README.md

```diff
@@ -1 +1,43 @@
-# covid_state_checker
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/89effbe5-ea91-4f9d-a00a-8fc5d63b390d/deploy-status)](https://app.netlify.com/sites/covid-state/deploys)
+# Space Traveler Hub
+This web app allows users to see available rockets for booking and missions they can join. Users can see their booked rockets and joined missions in the My-profile page and cancel any joined mission or booked rocket.
+
+
+# Built with
+- React.js
+- Redux
+- Tailwind
+
+## Live Demo
+https://covid-state.netlify.app
```

Comment:
- [ ] The live demo shows a blank page, Kindly fix the deployment of your app so that it works as expected
<img width="563" alt="chrome_spH131sgiO" src="https://user-images.githubusercontent.com/81468675/169603101-eb3934e9-242b-4e86-a5d3-d2282b734f0f.png">
.

Reply:
Well resolved 👍🏽